python/sglang/srt/distributed/device_communicators/all_reduce_utils.py

```diff
@@ -0,0 +1,16 @@
+MiB = 1024 * 1024
+
+SYMM_MEM_ALL_REDUCE_MAX_SIZES = {
+    9: {
+        2: 64 * MiB,  # 64 MB
+        4: 32 * MiB,  # 32 MB
+        6: 64 * MiB,  # 64 MB
+        8: 64 * MiB,  # 64 MB
+    },
+    10: {
+        2: 8 * MiB,  # 8 MB
```

Comment:
Why is Blackwell's TP2 configuration (8MB) smaller than Hopper's (64MB)? This seems unreasonable, please verify.

Reply:
Updated to 64MB for tp2 in Blackwell.